packages/fxa-content-server/tests/lib/restmail.js

```diff
@@ -4,8 +4,7 @@
 
 const request = require('./request');
 
-const config = intern._config;
-const EMAIL_SERVER_ROOT = config.fxaEmailRoot;
+const EMAIL_SERVER_ROOT = 'http://localhost:9001';
```

Comment:
Can you explain these changes?
So it looks like we deleted the intern.js config file above, so seems like we're now hardcoding this value. But&hellip;
```js
7: const args = require('yargs').argv;
29: const fxaEmailRoot = args.fxaEmailRoot || 'http://localhost:9001';
```

Do we need to account for an overridden fxaEmailRoot (or --fxa-email-root?) here? Looks like the packages/fxa-content-server/package.json file above deleted something with this line:

```js
"test-latest": "SKIP_MOCHA=true node tests/intern.js --fxaAuthRoot=https://latest.dev.lcip.org/auth/v1 --fxaContentRoot=https://latest.dev.lcip.org/ --fxaEmailRoot=http://restmail.net --fxaOAuthApp=https://123done-latest.dev.lcip.org/ --fxaUntrustedOauthApp=https://321done-latest.dev.lcip.org/ --fxaProduction=true --fxaToken=https://token.dev.lcip.org/1.0/sync/1.5",
```

So not sure if this should be `http://localhost:9001` or `https://restmail.net` (like it might have been in package.json)? Or maybe those configs were unused since they sound hella old ("lcip" being the inverted dev version of "picl", which i forget what that stood for); and https://latest.dev.lcip.org/ seems to throw a 503 anyways.

Reply:
I deleted the `intern.js` file where so the `restmail.js` file was giving error on the `intern` definition of:
```
const config = intern._config;    
const EMAIL_SERVER_ROOT = config.fxaEmailRoot;
```
So I hardcoded the value (http://localhost:9001/) for `EMAIL_SERVER_ROOT`  from the `intern.js` file